src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheck.java

```diff
@@ -126,48 +126,107 @@
  * public class TestOne
  * {
  *
  * }
  *
- * &#64;SomeArrays(pooches={DOGS.LEO}, um={}, test={"bleh"}) // Violation - COMPACT_NO_ARRAY
- * &#64;SuppressWarnings("") // OK
- * &#64;Deprecated() // Violation - cannot have closing parenthesis
+ * &#64;SuppressWarnings(value={"unchecked"}) // Violation - no parameter should be referenced
+ * &#64;Deprecated() // Violation - cannot have a closing parenthesis
+ * &#64;SomeArrays(value={"unchecked","unused",}) // Violation - cannot have a trailing array comma
  * class TestTwo {
  *
  * }
  * </pre>
  * <p>
  * To configure the check to enforce an {@code expanded} style,
- * with a trailing array comma set to {@code never}
- * and always including the closing parenthesis.
+ *             with a closing parenthesis and a trailing array comma set to {@code never}.
  * </p>
  * <pre>
  * &lt;module name=&quot;AnnotationUseStyle&quot;&gt;
- *   &lt;property name=&quot;elementStyle&quot; value=&quot;expanded&quot;/&gt;
- *   &lt;property name=&quot;trailingArrayComma&quot; value=&quot;never&quot;/&gt;
- *   &lt;property name=&quot;closingParens&quot; value=&quot;always&quot;/&gt;
+ *  &lt;property name=&quot;elementStyle&quot; value=&quot;expanded&quot;/&gt;
+ *  &lt;property name=&quot;closingParens&quot; value=&quot;never&quot;/&gt;
+ *  &lt;property name=&quot;trailingArrayComma&quot; value=&quot;never&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
  * Example:
  * </p>
  * <pre>
- * &#64;Deprecated // Violation - must have closing parenthesis
- * &#64;SomeArrays(pooches={DOGS.LEO}) // OK
- * &#64;SuppressWarnings({""}) // Violation - EXPANDED
+ * &#64;SuppressWarnings("unchecked") // Violation - parameters should be referenced
+ * &#64;Deprecated // OK
+ * &#64;SomeArrays({"unchecked","unused"}) // Violation - parameters should be referenced
+ * public class TestOne
+ * {
+ *
+ * }
+ *
+ * &#64;SuppressWarnings(value={"unchecked"}) // OK
+ * &#64;Deprecated() // Violation - cannot have a closing parenthesis
+ * &#64;SomeArrays(value={"unchecked","unused",}) // Violation - cannot have a trailing array comma
+ * class TestTwo {
+ *
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to enforce a {@code compact} style,
+ *             with always including a closing parenthesis and ignoring a trailing array comma.
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;AnnotationUseStyle&quot;&gt;
+ *  &lt;property name=&quot;elementStyle&quot; value=&quot;compact&quot;/&gt;
+ *  &lt;property name=&quot;closingParens&quot; value=&quot;always&quot;/&gt;
+ *  &lt;property name=&quot;trailingArrayComma&quot; value=&quot;ignore&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * &#64;SuppressWarnings("unchecked") // OK
+ * &#64;Deprecated // Violation - must have a closing parenthesis
+ * &#64;SomeArrays({"unchecked","unused"}) // OK
+ * public class TestOne
+ * {
+ *
+ * }
+ *
+ * &#64;SuppressWarnings(value={"unchecked"}) // Violation - 'value' should not be referenced
```

Comment:
`Violation - parameter 'value' should not be referenced` to both violation of this block